packages/nodes-base/nodes/Jira/JiraTrigger.node.ts

```diff
@@ -411,13 +417,29 @@ export class JiraTrigger implements INodeType {
 
 				const events = this.getNodeParameter('events') as string[];
 
-				const endpoint = '/webhooks/1.0/webhook';
-
-				const webhooks = await jiraSoftwareCloudApiRequest.call(this, endpoint, 'GET', {});
+				const version = await (
+					jiraSoftwareCloudApiRequest.call(
+						this,
+						'/api/2/serverInfo',
+						'GET',
+					) as Promise<JiraServerInfo>
+				)
+					.then((info) => info.versionNumbers?.[0] ?? 1)
+					.catch(() => 1);
+
+				const endpoint = version >= 10 ? '/jira-webhook/1.0/webhooks' : '/webhooks/1.0/webhook';
+				webhookData.endpoint = endpoint;
```

Comment:
Jira cloud returns:
```json
{
"baseUrl": "...",
"displayUrl": "...",
"displayUrlServicedeskHelpCenter": "...",
"displayUrlConfluence": "...",
"version": "1001.0.0-SNAPSHOT",
"versionNumbers": [
1001,
0,
0
],
"deploymentType": "Cloud",
"buildNumber": 100283,
"buildDate": "2025-03-31T22:18:01.000+0200",
"serverTime": "2025-04-02T03:21:54.857+0200",
"scmInfo": "fb04868edf98ed04012f265c2a91aefa6aaa0dd3",
"serverTitle": "Jira",
"defaultLocale": {
"locale": "en_GB"
},
"serverTimeZone": "GMT"
}
```

Therefor `version >= 10` is `true` and an unknown route would be used for cloud.
Maybe check `deploymentType` as well.

Reply:
Thanks @Fank! that makes sense, updated the implementation to consider Jira Cloud 🎉